pages/howto/containerization.md

```diff
@@ -23,17 +23,17 @@ Because Docker images are widely available for many software packages, a common
 
 ## Apptainer
 
-Apptainer is a containerization software package that does not require users to have administrative privileges when running containers, and can thus be safely used on Research Computing resources. Currently, an older version of singularity is available on compute nodes so the commands outlined here will use the `singularity` command as opposed to `apptainer`. 
+Apptainer is a containerization software package that does not require users to have administrative privileges when running containers, and can thus be safely used on Research Computing resources. Apptainer is functionally the same as Singularity and the commands can be used interchangeably at this time.  
 
 Much like Docker, Apptainer is a containerization software designed around compartmentalization of applications, libraries, and workflows. This is done through the creation of compressed images in the `.sif` format which can be run as ephemeral containers. Unlike Docker, however, Apptainer does not manage images, containers, or volumes through a central application. Instead, Apptainer generates saved image files that can either be mutable or immutable based on compression.
 
 ### Setting up Paths
-Images generated by Apptainer can be _large_. It is common for singularity containers to exceed 10GB. This means you can easily exceed your 10GB home directory quota while pulling a single container! You may also see error messages when pulling containers from remote repositories mentioning how `/tmp` does not have write permissions. All of these problems and more can be avoided by setting a few environmental variables before pulling apptainer containers.  Below is an example of how to set the environment variables.  You'll need to set the path name to your group's shared project directory, which may not be in `/projects/academic`, and make sure these directories are created:
+Images generated by Apptainer can be _large_. It is common for these containers to exceed 10GB. This means you can easily exceed your 25GB home directory quota while pulling a single container! You may also see error messages when pulling containers from remote repositories mentioning how `/tmp` does not have write permissions. All of these problems and more can be avoided by setting a few environmental variables before pulling Apptainer containers.  Below is an example of how to set the environment variables.  You'll need to set the path name to your group's shared project directory, which may not be in `/projects/academic`, and make sure these directories are created:
 
 ```
-export SINGULARITY_LOCALCACHEDIR=/projects/academic/[YourGroupName]/[CCRusername]/tmp
-export SINGULARITY_CACHEDIR=/projects/academic/[YourGroupName]/[CCRusername]/tmp
-export SINGULARITY_TMPDIR=/projects/academic/[YourGroupName]/[CCRusername]/tmp
+export APPTAINER_LOCALCACHEDIR=/projects/academic/[YourGroupName]/[CCRusername]/tmp
```

Comment:
Did Andrew say these are no longer necessary when he updated Apptainer versions a few downtimes ago?

Reply:
Good question.  I'm not sure.  @aebruno?

Reply:
Yes, these are all still required